.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
   - pwd
   - chmod +x ./scripts/ci/install_myhdl.sh
   - ./scripts/ci/install_myhdl.sh
+  - pip install -e git://github.com/cfelton/rhea.git#egg=rhea
```

Comment:
As we are installing MyHDL package from scripts, I just added the `RHEA` folder.